Auth/app/src/main/java/com/example/Auth/Requests.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.Auth
+
+import android.os.AsyncTask
+import android.util.Log
+import okhttp3.*
+
+class Requests {
+    internal class getUserProfileInfo(/*val url : String, val id : Int, val token : String*/) :
+        AsyncTask<Void, Void, String>() {
+
+        override fun doInBackground(vararg params: Void?): String {
+            val client = OkHttpClient()
+            val url = "http://10.0.2.2:8080/api/v1/profile/1/1"
+            val request = Request.Builder().url(url).addHeader("token", "111").build()
```

Comment:
Some of this code repeats multiple times, it can be moved to some method (something like prepareReuest(url: string) = Request.Builder().url(url).addHeader("token", "111"))